src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -73,6 +73,13 @@ public DisputesController(IBus bus, IMediator mediator, ILogger<DisputesControll
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public async Task<IActionResult> CreateAsync([FromBody] Models.Disputes.NoticeOfDispute dispute, CancellationToken cancellationToken)
     {
+        if (dispute.TicketNumber is not null && dispute.ViolationTicket is not null && dispute.ViolationTicket.TicketNumber is not null 
```

Comment:
Wondering if this logic couldnt be added to `NoticeOfDisputeValidator` ?

Reply:
Let's accept your change, but create another Jira issue to consolidate to the validator.

Reply:
Thanks for reminding the FluentValidation for that model. No problem, I can move this check to the validator before merging. Should not take much time.